assets/js/theme/account.js

```diff
@@ -105,7 +107,10 @@ export default class Account extends PageManager {
 
             if (!submitForm) {
                 event.preventDefault();
-                alert('Please select one or more items to reorder.');
+                swal({
```

Comment:
Can we have a intention-revealing function name? `swal` doesn't communicate anything.